sufficient_statistics.cc

```diff
@@ -162,6 +170,7 @@ double SufficientStatistics::GetSequencingErrorStatistic(const TrioModel &params
 }
 
 /**
+    //Note: Many duplicate code here, GetMismatches(const ReadDataVector &data_vec) can call GetMismatches(const ReadData &data). Same for heter/homo
```

Comment:
I removed the other overloaded functions like GetMismatches(const ReadData &data)